conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeOperator.scala

```diff
@@ -698,7 +697,7 @@ class TezosNodeSenderOperator(
     network: String,
     batchConf: BatchFetchConfiguration,
     sodiumConf: SodiumConfiguration,
-    apiOperations: ApiOperations
+    apiOperations: LorreOperations
```

Comment:
Same reasoning as above.

We could choose `apiOperations` in all places for this file, it makes some sense.